src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -10,13 +10,13 @@ import { ITerminalCapabilityStore, ITerminalCommand, TerminalCapability } from '
 import type { ITerminalAddon } from 'xterm-headless';
 import * as dom from 'vs/base/browser/dom';
 import { IContextMenuService } from 'vs/platform/contextview/browser/contextView';
-import { ICommandAction, ITerminalContextualActionOptions } from 'vs/workbench/contrib/terminal/browser/terminal';
+import { ICommandAction, ITerminalQuickFixActionOptions } from 'vs/workbench/contrib/terminal/browser/terminal';
 import { DecorationSelector, updateLayout } from 'vs/workbench/contrib/terminal/browser/xterm/decorationStyles';
 import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
 import { Terminal } from 'xterm';
 import { IAction } from 'vs/base/common/actions';
 
-export interface IContextualAction {
+export interface QuickFixAction {
```

Comment:
This is missing the `I` prefix, plus I think the replacement should be "contextual action" -> "quick fix", _fix_ is the action. Or "terminal quick fix" when we export anything so annoying imports don't show up when developing (editor) quick fixes

Reply:
yes agree, nice catch

Reply:
in some cases, `quick fix actions` actually make sense though https://github.com/microsoft/vscode/blob/4167f07e40f41e2d5b313b105ccbd7f972c324a7/src/vs/workbench/contrib/terminal/browser/terminalInstance.ts#L612-L617